nmdc_server/migrations/versions/988105f6581d_update_datasetDoi_to_awardDoi.py

```diff
@@ -0,0 +1,77 @@
+"""Update datasetDoi on context page
+
+Part of the ongoing additions to the submission portal includes updating
+the context form's datasetDoi to awardDoi. Doing this also changes the field to be an array/list.
+This migration will update the name of datasetDoi in existing entries to awardDoi, convert the field
+to a list/array, and update any values that need it to be list
+(such as comma delimited lists users have entered).
+
+Revision ID: 988105f6581d
+Revises: 5fb9910ca8e6
+Create Date: 2024-10-07 18:14:38.465850
+
+"""
+
+from typing import Optional
+from uuid import uuid4
+
+from alembic import op
+from sqlalchemy import Column, orm
+from sqlalchemy.dialects.postgresql import JSONB, UUID
+from sqlalchemy.ext.declarative import declarative_base
+
+Base = declarative_base()
+# revision identifiers, used by Alembic.
+revision: str = "988105f6581d"
+down_revision: Optional[str] = "5fb9910ca8e6"
```

Comment:
Again you can manually edit this line. No need to run a new migration.

Reply:
Gotcha, thanks Mary. I'll edit those.